### DISCUSSION TITLE
Not able to paste ssh key

### DISCUSSION THREAD

**Discussion by devops-4u** [community/community#32609] [New to GitHub]
Hii Team

I am not able to paste ssk key in github from git.
I tried alot but its not working.
I am able to copy and paste key eveywhere but not in github.
please help!

Regards,
Jatin

**Comment by queenofcorgis** [comment]
Hi Jatin! 

Have you added the SSH key to your GitHub account? 

[Here are the steps](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account), if you haven't done so.

**Comment by devops-4u** [comment]
Hi,

I find my key but i am not able to paste that key in github in add ssh key .

BR,
Jatin

**Reply by airtower-luna** [reply]
People can only help if you're a lot more specific about what you tried. Describe *exactly* what you do, and the results you get.

**Comment by devops-4u** [comment]
Hi,

I am not able to that key in key field.As i already copied my key (ctrl+c)
but its not pasting here.
Other than github it was showing option to paste but ij github its not
showing option to paste.
Ple help

**Comment by Shivansh-Mittal** [comment]
Hey, 
Don't watch or wait for paste option in key section.
Just click in the key box and when cursor start appearing, use `Ctrl + V` on keyboard. 
It works, for me & everyone and will work for you also

Have you generated ssh key successfully on your system ? If not then use `ssh-keygen` in your cmd terminal and follow the steps. And find ".ssh" folder in your computer and copy the contents of file with ".pub" extension.

**Comment by devops-4u** [comment]
ctrl+v also not working.

**Reply by DSDmark** [reply]
![Screenshot from 2022-09-23 09-17-10](https://user-images.githubusercontent.com/85744816/191888754-47b18e14-7b8b-4c80-8594-2c162927af84.png)
hey, how about open inspect element and try there to do past like that:-

**Comment by Liamss** [comment]
I had a similar problem at the start of a training course. It appeared as thought the prompt wasn't copying to clipboard. 

Initially I was using "clip <**>** ~/.ssh/id_ed25519.pub"

but I should have been using" clip < ~/.ssh/id_ed25519.pub" (without the second arrow) - then it worked for me.

**Comment by chriseggroll** [comment]
I had this same problem - could not paste my public key into the box.  I was using MS Edge, worked fine with Chrome, in case someone else stumbles on this.
